### PR TITLE
Typed default blocking dispatcher + docs

### LINKED ISSUE #24893: Typed, DispatcherSelector default for blocking()
It would make things easier for beginners if we'd provide a built in DispactherSelector.blocking() in addition to `default()`, otherwise there is the additional hoops to jump through to do the "right thing" ™.

Now:

```
/**
 * Factories for [[DispatcherSelector]]s which describe which thread pool shall be used to run
 * the actor to which this configuration is applied. Se the individual factory methods for details
 * on the options.
 *
 * The default configuration if none of these options are present is to run
 * the actor on the same executor as its parent.
 */
object DispatcherSelector {

  /**
   * Scala API:
   * Run the actor on the same executor as its parent.
   */
  def default(): DispatcherSelector = DispatcherDefault()

  /**
   * Java API:
   * Run the actor on the same executor as its parent.
   */
  def defaultDispatcher(): DispatcherSelector = default()

  /**
   * Look up an executor definition in the [[ActorSystem]] configuration.
   * ExecutorServices created in this fashion will be shut down when the
   * ActorSystem terminates.
   */
  def fromConfig(path: String): DispatcherSelector = DispatcherFromConfig(path)
}

```

Add this:

```
object DispatcherSelector {

  /**
   * Dispatcher suggested to be used for blocking operations.
   * You may also create dedicated dispatchers for specific groups of blocking operations etc.
   */
  def blocking(): DispatcherSelector = ???
}

```

### PR DESCRIPTION
**Pull request by chbatey**
Closes #24893

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/dispatchers.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Introduction 
+
+An Akka `MessageDispatcher` is what makes Akka Actors "tick", it is the engine of the machine so to speak.
+All `MessageDispatcher` implementations are also an @scala[`ExecutionContext`]@java[`Executor`], which means that they can be used
+to execute arbitrary code, for instance @ref:[Futures](../futures.md).
+
+## Selecting a dispatcher
+
+A default dispatcher is used for all actors that are spawned without specifying a customer dispatcher.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/dispatchers.md]: dispatch with the customers!

**Review by johanandren — APPROVED** [review]
LGTM with one nitpick

**Review by patriknw — APPROVED** [review]
LGTM, fix the customer and then merge

File: akka-docs/src/main/paradox/typed/dispatchers.md (hunk 1 above)
Comment (chbatey) [reply akka-docs/src/main/paradox/typed/dispatchers.md]: Doh lol

**Pull request merged**